sass/_navigation.scss

```diff
@@ -0,0 +1,156 @@
+.main-navigation {
+    
+    &__list {
+        list-style: none;
+        
+        display: flex;
+        justify-content: space-around;
+    }
+    
+    &__item {
+        text-transform: uppercase;
+        font-size: 1.6rem;
+        padding: 2px 0;
+        position: relative;
+        
+        &:not(:last-child) {
+            margin-right: 2rem;
+            
+            @media(max-width: 480px) {
+                margin-right: 1rem;
+            }
+        }
+                
+        &::before {
+            content: "";
+            position: absolute;
+            top: 100%;
+            left: 0;
+            width: 100%;
+            height: 2px;
+            background-color: $color-secondary;
+            transform: scaleX(0);
+            transition: all .4s;
+        }
+        
+        &:hover {
+            cursor: pointer; 
+         
+            &::before {
+                transform: scaleX(1.1);
+            }
+        }
+    }
+}
+
+.side-navigation {
```

Comment:
If you change the HTML part of your .side-navigation, you will have to rewrite css too.